main/lsp/lsp.cc

```diff
@@ -31,10 +32,15 @@ void LSPLoop::sendCountersToStatsd(chrono::time_point<chrono::steady_clock> curr
     Timer timeit(config->logger, "LSPLoop::sendCountersToStatsd");
     ENFORCE(this_thread::get_id() == mainThreadId, "sendCounterToStatsd can only be called from the main LSP thread.");
     const auto &opts = config->opts;
-    // Record rusage-related stats.
-    StatsD::addRusageStats();
     auto counters = getAndClearThreadCounters();
     if (!opts.statsdHost.empty()) {
+        // Record rusage-related stats.
+        StatsD::addRusageStats();
+        // Record version information.
+        prodCounterAdd("release.build_scm_commit_count", Version::build_scm_commit_count);
```

Comment:
Could 
https://github.com/sorbet/sorbet/blob/7d705e45/main/realmain.cc#L87
be called/extracted/reused instead?
Would love to not have >1 place where metric name can be misspelled.

Reply:
Could I put it into `StatsD` and add a dependency between `Statsd` and `Version`?